nixops/backends/__init__.py

```diff
@@ -237,7 +257,9 @@ def send_keys(self):
               )
             )
             os.remove(tmp)
-        self.run_command("touch /run/keys/done")
+        self.run_command("mkdir -m 0750 -p /run/keys && "
+                         "chown root:keys  /run/keys && "
```

Comment:
Ok, makes sense. Although this known location is _only_ used in https://github.com/NixOS/nixops/blob/master/nix/keys.nix (needs update as well). And for me the most wanted consequence of this is reboot-able servers, which do not lose keys after reboot. If any service is bound to "old" `/run/keys/done` it is a bug (or a DRY sin).